src/gen/lib/Codegen_parse.ml

```diff
@@ -10,6 +10,13 @@ open Indent.Types
 
 let debug = false
 
+(* All rule names and other names directly defined in grammar.json
+   must go through this translation. For example, it turns
+   "true" into "true_" because "true" is a reserved keyword in the generated
+   code.
+*)
+let trans = Codegen_util.translate_ident
```

Comment:
Fixes name clashes we ran into when testing on the Ruby grammar e.g. `method` is a rule name that can't be used as an OCaml type. This was done (for Reason) for generating AST.ml but not for Parse.ml. It's clean now.